modules/ebi-metagenomics/taxonkit/reformat/main.nf

```diff
@@ -0,0 +1,52 @@
+process TAXONKIT_REFORMAT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/taxonkit:0.17.0--h9ee0642_1':
+        'biocontainers/taxonkit:0.17.0--h9ee0642_1' }"
+
+    input:
+    tuple val(meta), path(tsv)
+    path taxdb
+
+    output:
+    tuple val(meta), path("*.tsv"), emit: tsv
```

Comment:
I think you should rename this from `tsv` since the input is called `tsv` already, maybe `reformat_tsv` or `out_tsv` or something

Reply:
`reformat_tsv` is sensible